Watchman.Integrations/Disboard/BumpedChannel.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+
+namespace Watchman.Integrations.Disboard
+{
+    class BumpedChannel
+    {
+        public ISocketMessageChannel MessageChannel { get; }
+        
+        private const string _bumpMessage = "!d bump";
+
+        private static TimeSpan HowOften => new TimeSpan(2, 1, 0); // every 2 hours and 1 minute
```

Comment:
```suggestion
        private static TimeSpan HowOften = new TimeSpan(2, 1, 0); // every 2 hours and 1 minute
```

Reply:
chcemy to ustawiać raz przy tworzeniu obiektu 
a nie przy każdym użyciu 

+ dodaj readonly